.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,5 @@
+Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request.
+
+If it fixes a bug or resolves a feature request, be sure to link to that issue.
+
+The best way to propose a feature is to open an issue first and discuss your ideas there before implementing them.
```

Comment:
Ideas don't have to be long RFCs, rather opener discussions on a certain suggestion (if said suggestion would take much more time than opening/discussing it for a few days). Otherwise, I agree with @dereuromark, in all other cases, a quick PR is encouraged.

Am just not sure how to best word it in the template. I'll leave that to you @thinkingmedia :wink: